gobblin-core/src/main/java/gobblin/writer/commands/JdbcWriterCommands.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.writer.commands;
+
+import gobblin.converter.jdbc.JdbcType;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Map;
+
+public interface JdbcWriterCommands extends JdbcBufferedInserter {
```

Comment:
Will do. (It's basically interface to each RDBMS as each RDBMS may uses different SQL syntax)